### PR TITLE
init add of pytut by @orza

### PR DESCRIPTION
**Pull request by arkadianriver**
@orzade , your tutorial with minor alignments to markdown.
To review this pull request without git:
1. Unzip the this branch somewhere: https://github.com/arkadianriver/streamsx.documentation/archive/orza-pytut.zip
2. In the streamsx.documentation folder where you unzipped it, run `jekyll serve`

### CODE REVIEW

**Comment by orzade** [comment]
Thanks, @arkadianriver. It looks good to me

**Review by jakrmpotic — COMMENTED** [review]
@orzade @arkadianriver there's a word missing in step 4 "Enter the name of your service and define the build configuration used deploy your app to the service:".  Add "to" between "used deploy"?

**Comment by arkadianriver** [comment]
ko, fixed, @jakrmpotic. Shall we merge you think? Any more reviewers for this content?

**Comment by jakrmpotic** [comment]
@arkadianriver - We can merge once @orzade confirms that review comments were already addressed and she's not waiting for more reviews.

**Comment by orzade** [comment]
@arkadianriver @jakrmpotic I confirm that all review comments have been addressed. I'm not waiting for more reviews. Thanks

File: docs/python/tutorial_python.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+published: true
+---
+
+You can create an app in your Python environment, and then deploy the app to an instance of the Streaming Analytics service running on IBM Bluemix.
+
+The Streaming Analytics service is built on IBM Streams technology. You do not need a local version of IBM Streams to build apps for the service.
+
+This tutorial shows how to build a simple app with Python. The app runs as a job in your Streaming Analytics instance. 
+
+*Note:* Familiarity with Python is recommended. This tutorial is designed to run in a Python 3.5 environment.
```
Comment (ddebrunner) [inline docs/python/tutorial_python.md]: It requires a 3.5 environment.

File: docs/python/tutorial_python.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+The following steps show how to set up your Python environment and your Streaming Analytics instance:
+
+1. Install [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and set the JAVA_HOME environment variable:
+   ```
+   export JAVA_HOME="/usr/lib/jvm/java-1.x.x-openjdk"
+   export PATH=$JAVA_HOME/bin:$PATH
+   ```
+2. Install [Anaconda 3](https://www.continuum.io/downloads). Follow the instructions that apply to your operating system.
+
+3. Set the bin and scripts directories in the `PATH` environment variable:
```
Comment (ddebrunner) [inline docs/python/tutorial_python.md]: Doesn't the Ananconda install do this for you?

File: docs/python/tutorial_python.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+   source activate py35
+   ```
+6. [Install the Python package pypi.streamsx](#step1)
+7. [Start a Streaming Analytics service](#step2)
+8. [Set up access to the service](#step3)
+
+<a id="step1"/>
+### Installing the Python package pypi.streamsx
+
+Install the latest Python package pypi.streamsx from the Python Package Index (PyPI) into your Python environment:
```
Comment (ddebrunner) [inline docs/python/tutorial_python.md]: The package is `streamsx`, not `pypi.streamsx`.

File: docs/python/tutorial_python.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+   Paste your credentials using the following command:
+
+   ```python
+   c={
+     # ... *paste your credentials here*
+   }
+   ```
+
+4. Create a simple Hello World! streaming app. To create a simple streaming application you must first create a Topology object. A topology object represents a declaration of a streaming application that will be executed by a Streams instance as a job. The topology also specifies an iterable stream source and prints the contents of the stream to the application console:
```
Comment (ddebrunner) [inline docs/python/tutorial_python.md]: Awkward last sentence. There is a sudden switch what from a Topology generally is to what this specific application is doing.

File: docs/python/tutorial_python.md (hunk 2 above)
Comment (orzade) [reply docs/python/tutorial_python.md]: At least in Linux and Windows environments, I had to do it manually.

File: docs/python/tutorial_python.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+You can create an app in your Python environment, and then deploy the app to an instance of the Streaming Analytics service running on IBM Bluemix.
+
+The Streaming Analytics service is built on IBM Streams technology. You do not need a local version of IBM Streams to build apps for the service.
+
+This tutorial shows how to build a simple app with Python. The app runs as a job in your Streaming Analytics instance. 
+
+*Note:* Familiarity with Python is recommended. This tutorial is designed to run in a Python 3.5 environment.
+
+<!-- This set up section will probably be removed when the DSX integration with the Streaming Analytics service is enhanced -->
```
Comment (ddebrunner) [inline docs/python/tutorial_python.md]: Why would this section be removed?

This is not a DSX tutorial so any changes in DSX do not affect this.

**Pull request merged**